### PR TITLE
feat(mysql-setup): Add the ability to specify database name for mysql-setup

### PR DESCRIPTION
**Pull request by dexter-mh-lee**
Add the ability to specify database name for mysql-setup
Does a simple string replace on the init.sql to change the db name. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docker/mysql-setup/init.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sed -i -e "s/DATAHUB_DB_NAME/${DATAHUB_DB_NAME}/g" /init.sql
+cat /init.sql
```
Comment (jjoyce0510) [inline docker/mysql-setup/init.sh:4]: necessary?

**Review by jjoyce0510 — APPROVED** [review]
lgtm¡

File: docker/mysql-setup/init.sh (hunk 1 above)
Comment (dexter-mh-lee) [reply docker/mysql-setup/init.sh]: removed!

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**